### DISCUSSION TITLE
The framework 'Microsoft.NETCore.App', version '6.0.6' (x64) was not found

### DISCUSSION THREAD

**Discussion by ayankumaar** [getgauge/gauge#2258] [General]
We downloaded the latest gauge dotnet (0.5.0) plugin around a month back. It was working fine with .NET SDK 6.0.300 and runtime 6.0.5. However a few days back we started to get an error in the lsp.log file that **The framework 'Microsoft.NETCore.App', version '6.0.6' (x64) was not found**

It looks like it is expecting .NET SDK 6.0.301 which contains the runtime 6.0.6.

I am trying to understand what is the reason behind this auto upgrade ? Do we need to always keep updating to the latest .NET SDK for gauge ?

**Comment by ayankumaar** [comment]
@sriv or @zabil any update on this query ?

**Reply by ayankumaar** [reply]
Can anybody please answer on this ? Every few weeks I am installing a new .NET SDK/Runtime otherwise gauge is not working. Is it an expected behavior ?

**Reply by zabil** [reply]
Can you share the exact [log](https://docs.gauge.org/faqs.html?os=macos&language=javascript&ide=vscode#logs) messages? (with any sensitive information redacted). 

I Don't use dotnet setup but there's nothing in the code that checks for the latest version.
